### PR TITLE
Parallelization

### PR DESCRIPTION
**Pull request by austinmarckx**
Added force parallelization code

### CODE REVIEW

**Comment by austinmarckx** [comment]
Updated the Regression unit tests

**Review by XinxinWangUTSW — APPROVED** [review]
Profiler nicely done.

File: TestRBC.m [hunk 1]
```diff
@@ -153,60 +165,57 @@ function testVolumeSyntaxAlternative(testCase)
...
         
         % Unit test: DeltaHFromIdeal
         function testDeltaHFromIdeal(testCase)
+            import matlab.unittest.constraints.IsEqualTo
+            import matlab.unittest.constraints.RelativeTolerance
            % make sure 
-           deltaH = exists(deltaHFromIdeal(testCase.m, testCase.m.pm.k_c));
-           testcase.verifyEqual(deltaH, 1, 'deltaHFromIdeal Exists')
+           deltaH = deltaHFromIdeal(testCase.m_new, testCase.m.pm.k_c);
+           testCase.verifyThat(deltaH(1:5), IsEqualTo(testCase.ExpectedDeltaH, 'Within', RelativeTolerance(0.05)), 'deltaHFromIdeal Exists')
```
Comment (andrewjUTSW) [inline TestRBC.m:179]:
```suggestion
           testCase.verifyThat(deltaH(1:5), IsEqualTo(testCase.ExpectedDeltaH, 'Within', RelativeTolerance(0.05)), 'deltaH is within Tolerance')
```

**Review by andrewjUTSW — APPROVED** [review]
Looks better, thanks!  Could continue to add some more parameter configurations for regression tests if desired. Nice work!

**Pull request merged**